code/Rounds/BaseRound.cs

```diff
@@ -8,6 +8,7 @@ namespace ssl.Rounds
 {
     public abstract partial class BaseRound : NetworkComponent
     {
+        public event RoundEndedEvent RoundEndedEvent;
         public HashSet<MainPlayer> Players = new();
```

Comment:
Je trouve ça mieux de séparer par un retour à la ligne les events, tu me dis ce que t'en penses
```suggestion
        public event RoundEndedEvent RoundEndedEvent;
        
        public HashSet<MainPlayer> Players = new();
```